curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/write-higher-order-arrow-functions.english.md

```diff
@@ -71,16 +104,20 @@ console.log(squaredIntegers);
 <section id='solution'>
 
 ```js
-const realNumberArray = [4, 5.6, -9.8, 3.14, 42, 6, 8.34, -2];
```

Comment:
The solution still needs to have the realNumberArray defined on this line, so it can be referenced in the `squareList` function call.
```suggestion
const realNumberArray = [4, 5.6, -9.8, 3.14, 42, 6, 8.34, -2];
```

Reply:
I didn't mean to undo the map link. I guess I need to move slower when working on this stuff and check things over a day later, for mistakes. I'm not sure what I should do now. I can revert to the previous commit where they those files are untouched and re-commit.

I'll wait this time until I hear what U think I should do.

Reply:
If you add back the change I suggest above, I believe that will fix the failing test you were seeing.

Reply:
Hi @RandellDawson I made the change U suggested and it works. Stupid mistake on my part. 

Hopefully I can do this  process cleaner next time around. 

Are there any other changes to make currently?

Reply:
I assume you will just reject the changes to the guide files that were erroneous? That's what I was referring to  in the block comment above.

Reply:
@chrisdel101 You are not seeming to understand something.  We can not just reject parts of the PR and accept other parts.  We either accept or reject all the parts.  You still need to add:

You have a couple new things you need to address now with your latest commits.  First, you need to exclude the `package.json`  and the two `package-lock.json` files, so they do not appear in this PR.  Also, `the `guide/english/javascript/es6/map-function/index.md` needs to be reverted back before any changes, because the changes you are trying to make are not needed at all now, due to another PR which was merged yesterday.  If you can take care of these things and push the commits back up, that should be good enough to merge approve this PR.

Reply:
Do I need to remove them from the commit history and can I just remove from the latest version and recommit.
I am getting `noop` in all my attempts to `git rebase` the errors out.

Reply:
Also I only see one package.json lock on the root. Do U mean the ones in the folders as well? i.e. client?

Reply:
@chrisdel101 Yes, if you look at my previous comment, none of the package.json or package-lock.json files should be included as changes to your PR.

Also, you did not revert the changes to  `guide/english/javascript/es6/map-function/index.md`.  You deleted the MDN link instead of just not making any changes to that file.  Just leave that file as it originally was before you started making changes to it.

Reply:
@chrisdel101 You were not supposed to delete the package.json or the package-lock.json, but instead you were just supposed to leave them off this PR totally by reverting to how they were before you started making changes on this PR.

Reply:
I see all the commits in the commit history. When I try to rebase I have to resolve all the conflicts with everyone's changes. There are too many files to do this (like over 100) Is this what U meant by before?

Reply:
I reverted the files back what they were when I began working on this issue.